pom.xml

```diff
@@ -612,6 +614,9 @@
           <showDeprecation>true</showDeprecation>
           <contextPath>/jenkins</contextPath>
           <!-- TODO specify ${java.level} when JENKINS-20679 implemented -->
+          <systemProperties>
+            <hudson.Main.development>true</hudson.Main.development>
```

Comment:
Did you test this?

I think you want to replace the `true` with `${hudson.Main.development}`